scripts/set_plugin_version.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+PLUGIN_FILE_PATH="../crates/snforge-scarb-plugin/Scarb.toml"
+
+VERSION=$(cat "$PLUGIN_FILE_PATH" | grep version | cut -d '"' -f 2)
```

Comment:
useless use of cat

https://www.shellcheck.net/wiki/SC2002

Reply:
This probably explains it better
https://en.wikipedia.org/wiki/Cat_(Unix)#Useless_use_of_cat

Reply:
Cool, removed